de.wwu.md2.framework/src/de/wwu/md2/framework/validation/ModelValidator.xtend

```diff
@@ -64,6 +65,32 @@ class ModelValidator extends AbstractMD2JavaValidator {
             warning("Entity and Enum identifiers should start with an upper case letter", MD2Package.eINSTANCE.modelElement_Name, -1, ENTITYENUMUPPERCASE);
         }
     }
+    
+    /**
+     * Prevent from using an underscore "_" in the beginning of an entity name
+     */
+     @Check
+     def checkEntityDoesntStartWithUnderscore(ModelElement modelElement){
+     	if(modelElement.name.charAt(0).equals(new Character ('_'))){
+     		error("Entity and Enum identifiers shouldn't start with an underscore.",MD2Package.eINSTANCE.modelElement_Name,ENTITYWITHOUTUNDERSCORE);
+     	}
+     }
+     
+     /**
+     * Prevent from using the following preset identifiers as entity / enum names:
+     * - WorkflowState
+     */
+     @Check
+     def checkEntityNameDoesntEqualPresetIdentifiers(ModelElement modelElement){
+     	// Add new preset identifiers to the List, if they should also be excluded within the entity / enum names
+     	var presetIdentifiers = newArrayList("WorkflowState")
+     	// for every preset identifier, check if used as entity / enum name --> then error
+		for (identifier : presetIdentifiers ){
+			if(modelElement.name == identifier){
+     			error(identifier+" shouldn't be used as an entity / enum name, since it is a preset identifier.", MD2Package.eINSTANCE.modelElement_Name);
```

Comment:
Shouldn't this also get an error constant? Something like ENTITYWITHRESERVEDNAME?
Moreover, this validator needs to be tested.